src/Tools/dotnet-format/README.md

```diff
@@ -0,0 +1,48 @@
+﻿dotnet-format
+=============
+`dotnet-format` is a code formatter for `dotnet` that reads style preferences from your `.editorconfig` file and applies them to a project or solution.
+
+### How To Use
+
+The command will search the current directory for a project or solution file when a workspace isn't specified.
+
+    Usage: dotnet format [options] [[--] <args>...]
+
+    Options:
+      -?|-h|--help                Show help information
+      -w|--workspace <workspace>  When specified the project or solution file will be used as the workspace
+      -q|--quiet                  Suppresses all output except warnings and errors
+      -v|--verbose                Show verbose output
+
```

Comment:
Did you try having the parser create help for you? If you found issues, let us know. If not, you can get the text for the first part and append your examples.